recipes/cairomm/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# FIXME: This is a hack to make sure the environment is activated.
+# The reason this is required is due to the conda-build issue
+# mentioned below.
+#
+# https://github.com/conda/conda-build/issues/910
+#
+source activate "${CONDA_DEFAULT_ENV}"
+
+# Setup the building
+./configure --prefix=${PREFIX}  || { cat config.log; exit 1; }
```

Comment:
Probably need a flag like `--enable-static`. Would add `--enable-shared` for good measure.